fabulaws-config.yml

```diff
@@ -94,7 +94,7 @@
     - floridaopendebate.com
     - www.floridaopendebate.com
     staging:
-    -
+    - staging.demopenquestions.com
     testing:
     - testing.demopenquestions.com
```

Comment:
Let's move 'demopenquestions.com' to the top here, so that SITE_DOMAIN will use that for tweet urls, etc.